infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -31,6 +34,11 @@ export class AtatWebApiStack extends cdk.Stack {
       value: table.tableName,
     });
 
+    const submitQueue = new sqs.Queue(this, "SubmitQueue", {
+      queueName: "SubmitQueue.fifo",
+      fifo: true,
+    });
+
```

Comment:
Can you look at creating a version of this in `compliant-resources` now so that we don't have to handle that refactoring later? Even if it's not secure right now?